action.yml

```
@@ -0,0 +1,25 @@
+name: 'Appreciation Engine whitelist'
+description: 'Whitelists or unwhitelists domains for appreciation engine'
+inputs:
+  operation: 
+    description: 'whitelist or unwhitelist'
+    required: true
```

Make the default action whitelist and does not need to be passed in.

```suggestion
    required: false
```